src/main/java/com/fasttime/domain/resume/entity/Resume.java

```diff
@@ -0,0 +1,65 @@
+package com.fasttime.domain.resume.entity;
+
+import com.fasttime.domain.member.entity.Member;
+import com.fasttime.global.common.BaseTimeEntity;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import java.time.LocalDateTime;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.ColumnDefault;
+
+@Entity
+@Getter
+@AllArgsConstructor
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Builder
+public class Resume extends BaseTimeEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String title;
+    private String content;
+    @ColumnDefault("0")
+    private int likeCount;
```

Comment:
필드 간 줄 띄움을 맞추면 더 좋을 것 같습니다 ㅎㅎ
```suggestion
    private String title;

    private String content;

    @ColumnDefault("0")
    private int likeCount;
```